apps/versions/models.py

```diff
@@ -244,8 +244,8 @@ def compatible_platforms(self):
         The result is based on which app(s) the version targets.
         """
         app_ids = [a.application for a in self.apps.all()]
-        targets_mobile = amo.MOBILE.id in app_ids
-        targets_other = any((id_ != amo.MOBILE.id) for id_ in app_ids)
+        targets_mobile = amo.ANDROID.id in app_ids
```

Comment:
It's interesting that this didn't cause problems before... From what I can tell, it should have only made it possible to upload files for Android Firefox if they also claimed to support XUL Fennec, or specified a platform in the XPI.